pose_parameters:
  t_pipe_u: {t_vec: [[14], [20], [-180]], t_vec_up: [[14], [20], [-280]], r_vec: [[0], [0], [0]]}
  pipe_u: {t_vec: [[20], [27.5], [-165]], t_vec_up: [[20], [27.5], [-280]], r_vec: [[0], [0], [1.5708]]}
  L_pipe_u: {t_vec: [[60], [40], [-185]], t_vec_up: [[60], [40], [-280]], r_vec: [[0], [0], [0]]}
  T_pipe: {t_vec: [[14], [20], [-165]], t_vec_up: [[14], [20], [-280]], r_vec: [[0], [0], [0]]}
  t_pipe_up: {t_vec: [[45], [265], [20]], t_vec_up: [[45], [390], [20]], r_vec: [[1.5708], [0], [0]]}
  t_pipe_down: {t_vec: [[60], [25], [-185]], t_vec_up: [[60], [40], [-280]], r_vec: [[0], [0], [0]]}
